src/main/java/it/gov/pagopa/onboarding/workflow/service/OnboardingServiceImpl.java

```diff
@@ -319,38 +323,27 @@ private void checkStatus(Onboarding onboarding){
             !OnboardingWorkflowConstants.REJECTION_REASON_BIRTHDATE_KO.equals(onboarding.getDetailKO())){
       auditUtilities.logOnboardingKOWithReason(onboarding.getUserId(), onboarding.getInitiativeId(), onboarding.getChannel(),
               utilities.getMessageOnboardingKO(onboarding.getDetailKO()));
-      throw new OnboardingWorkflowException(HttpStatus.FORBIDDEN.value(),
-              utilities.getMessageOnboardingKO(onboarding.getDetailKO()),
-              onboarding.getDetailKO());
+      utilities.getOnboardingException(onboarding.getDetailKO(), onboarding.getInitiativeId());
```

Comment:
may you find a better semantic here? For me it's not clear this method throws exceptions.